pkg/services/orgresolver/linking.go

```diff
@@ -0,0 +1,116 @@
+package orgresolver
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+
+	log "github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	linkingclient "github.com/smartcontractkit/chainlink-protos/linking-service/go/v1"
+)
+
+// OrgResolver interface defines methods for resolving organization IDs from workflow owners
+type OrgResolver interface {
+	services.Service
+	Get(ctx context.Context, owner string) (string, error)
+}
+
+type Config struct {
+	URL                           string
+	TLSEnabled                    bool
+	WorkflowRegistryAddress       string
```

Comment:
Should/must this be in any particular format? e.g. hex? leading 0x? checksummed?
Sometimes we use another type backed by string (like `libocr.Account`) so it can be documented once, instead of all the different uses.